src/sidebar/util/search-filter.js

```diff
@@ -134,7 +134,7 @@ export function toObject(searchText) {
  * Terms that are not associated with a particular facet are stored in the "any"
  * facet.
  *
- * @param {string} searchText - Filter query to parse
+ * @param {string|null} searchText - Filter query to parse
```

Comment:
Necessary because of type definition of `ThreadState`

Reply:
Alternatively you could use the `|| fallback` pattern at the call site:

```diff
diff --git a/src/sidebar/util/root-thread.js b/src/sidebar/util/root-thread.js
index 17c7badd0..874530312 100644
--- a/src/sidebar/util/root-thread.js
+++ b/src/sidebar/util/root-thread.js
@@ -47,7 +47,7 @@ function buildRootThread(threadState) {
 
   if (annotationsFiltered) {
     const filters = generateFacetedFilter(
-      selection.filterQuery,
+      selection.filterQuery || '',
       selection.filters
     );
     options.filterFn = ann => filterAnnotations([ann], filters).length > 0;
diff --git a/src/sidebar/util/search-filter.js b/src/sidebar/util/search-filter.js
index cf8e74c5e..2b8627d40 100644
--- a/src/sidebar/util/search-filter.js
+++ b/src/sidebar/util/search-filter.js
@@ -134,7 +134,7 @@ export function toObject(searchText) {
  * Terms that are not associated with a particular facet are stored in the "any"
  * facet.
  *
- * @param {string|null} searchText - Filter query to parse
+ * @param {string} searchText - Filter query to parse
  * @param {FocusFilter} focusFilters - Additional filter terms to mix in
  * @return {Object.<string,Facet>}
  */
```

I have a mild preference for the above because it means there is only one way to represent an empty search when calling `searchText`, assuming you don't change how `generateFacetedFilter` handles an empty string.